README.adoc

```diff
@@ -34,6 +34,7 @@ buildPlugin(
 * `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
 * `useContainerAgent` (default: `false`) - uses a link:https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#container-agents[Container agent] instead of a Virtual Machine: usually faster to start and generates less costs for the project
 ** Please note that the implementation of "containers" can be changed over time
+* `forkCount` (default: null) - Execute tests with `forkCount` Java virtual machines. Common values include `0.45C`, `1`, `2`, and `1C` (if you terminate the value with a 'C', that value will be multiplied by the number of available CPU cores).
```

Comment:
My local setting of `0.45C` is probably inappropriate for a CI machine. I chose this specifically because I do _not_ want a desktop OS to be entirely consumed with tests since it can cause GUI applications to freeze up.